proto-google-cloud-bigtable-v2/src/main/java/com/google/bigtable/v2/RowFilterOrBuilder.java

```diff
@@ -18,10 +18,13 @@
 
 package com.google.bigtable.v2;
 
+import java.io.Serializable;
+
 public interface RowFilterOrBuilder
     extends
-    // @@protoc_insertion_point(interface_extends:google.bigtable.v2.RowFilter)
-    com.google.protobuf.MessageOrBuilder {
+        // @@protoc_insertion_point(interface_extends:google.bigtable.v2.RowFilter)
```

Comment:
unfortunately anything in the `proto-*` or `grpc-*` modules are auto generated and cannot be modified by hand.

Reply:
thanks, the changes are rolled back.